src/lib/components/NglMoleculeViewer.react.js

```diff
@@ -0,0 +1,286 @@
+import React, {Component} from 'react';
+import PropTypes from 'prop-types';
+import {Stage, Selection} from 'ngl';
+import * as R from 'ramda';
```

Comment:
Due to differences in project configurations and ramda versions behaving differently for default import, the preferred import/usage/syntax for Dash core packages is:
```
import { equals } from 'ramda';
```